### PR TITLE
fix: Various fixes

### PR DESCRIPTION
**Pull request by Mathematti**

### CODE REVIEW

**Comment by Mathematti** [comment]
@Ixpiria or @EinSev please fix conflicts

File: frontend/pages/journey/[id].vue [hunk 1]
```diff
@@ -84,10 +84,16 @@ useHead({
 });
 
 const colorMode = useColorMode();
+const darkThemeMq = window.matchMedia("(prefers-color-scheme: dark)");
+console.log(darkThemeMq);
+console.log(colorMode.preference);
```
Comment (Mathematti) [inline frontend/pages/journey/[id].vue]: Remove console.log

File: frontend/pages/journey/[id].vue [hunk 2]
```diff
@@ -84,10 +84,14 @@ useHead({
 });
 
 const colorMode = useColorMode();
+const darkThemeMq = window.matchMedia("(prefers-color-scheme: dark)");
 let darkColor = "#333333";
```
Comment (EinSev) [inline frontend/pages/journey/[id].vue]: We could improve this and don't hardcode this

**Review by EinSev — DISMISSED** [review]
and where is the calendar gap fix

File: frontend/pages/journey/[id].vue (hunk 2 above)
Comment (Mathematti) [reply frontend/pages/journey/[id].vue]: then do it

Comment (EinSev) [reply frontend/pages/journey/[id].vue]: do it urself

Comment (Mathematti) [reply frontend/pages/journey/[id].vue]: I don't know how, I'm just a backend worker

**Review by stefll — APPROVED** [review]
Design Review: Looks fine

**Review by Ixpiria — APPROVED** [review]

**Pull request merged**